pkg/webhook/webhook.go

```diff
@@ -211,3 +208,19 @@ func (pwh *bookkeeperWebhookHandler) InjectScheme(s *runtime.Scheme) error {
 	pwh.scheme = s
 	return nil
 }
+
+func CreateVersionMap() (map[string]string, error) {
```

Comment:
please create a private method with starting letter in small case.

Reply:
done